tools/src/main/python/dlpx/virtualization/_internal/validation_schemas/plugin_config_schema.json

```diff
@@ -54,6 +54,10 @@
       "luaName":  {
           "type": "string",
           "pattern": "^[a-z0-9_:-]+$"
+      },
+      "minimumLuaVersion": {
+          "type": "string",
+          "pattern": "^([0-9]+)\\.([0-9]+)\\.([a-zA-Z0-9_]+)$"
```

Comment:
I'd suggest using just the major/minor version here, and not the patch level. Something like `"^[0-9]+\\.[0-9]+$"`. Two reasons for this:
1) This is what we do with Lua already -- a Lua toolkit just says "I can upgrade from 3.2", not "I can upgrade from 3.2.5"
2) The patch level will need to be ignored by the engine anyhow (there is no ordering for patch levels, so it's impossible to determine a "minimum" patch version). If we include it here, we might cause plugin developers to mistakenly infer that the patch level matters here.

Reply:
Discussed with Lindsey in the Slack group chat, but I am convinced and agree that it should just be `major.minor` version, especially since the stack ignores the patch version.